### ISSUE TITLE
[cloud-tensorflow-mnist]:

### ISSUE THREAD

**Issue by jeromelebel** [googlecodelabs/feedback#600]
Hello,

In https://codelabs.developers.google.com/codelabs/cloud-tensorflow-mnist/#5

This line:
a,c = sess.run([accuracy, cross_entropy], feed=test_data)

Should be changed in:
a,c = sess.run([accuracy, cross_entropy], feed_dict=test_data)

**Comment by martin-gorner** [comment]
Thank you for your feedback. The entire lab was revamped for modern Tensorflow and now uses Keras as its high-level API. So no more sessions!
